undertow/src/main/java/org/wildfly/extension/undertow/Namespace.java

```diff
@@ -48,12 +48,13 @@
     UNDERTOW_10_0("urn:jboss:domain:undertow:10.0"),
     UNDERTOW_11_0("urn:jboss:domain:undertow:11.0"),
     UNDERTOW_12_0("urn:jboss:domain:undertow:12.0"),
+    UNDERTOW_12_1("urn:jboss:domain:undertow:12.1"),
     ;
 
     /**
      * The current namespace version.
      */
-    public static final Namespace CURRENT = UNDERTOW_12_0;
+    public static final Namespace CURRENT = UNDERTOW_12_1;
```

Comment:
Historically we have been incrementing only the major version number for quite a while. It is intersting that you brought up this, though, because I see that Undertow has 1.1, 1.2 and 3.1. @bstansberry do we have any policies in WF as to when should we upgrade the major model version versus the minor one?

Reply:
@fl4via @baranowb We should always do a major bump in WildFly main / WildFly Core main. That eliminates varying interpretations of what's 'minor' and leaves minor versions available for use in other streams if necessary.

I've never gotten a sense that people are trying to get semantic meaning from our API/schema versions so there's not much downside to just doing majors.

Reply:
@bstansberry roger.